.rubocop.yml

```diff
@@ -25,6 +25,10 @@ Style/Documentation:
 Style/HashSyntax:
   Enabled: false
 
+# https://github.com/bbatsov/rubocop/issues/3285
```

Comment:
This doesn't appear relevant - there aren't any frozen string literal comments in our files, so I don't see that it's a bug in rubocop. It's raising correct warnings.